docs/search/query-assist.md

```diff
@@ -0,0 +1,87 @@
+---
+id: query-assist
+title: Logs Query Assist - Preview
+description: Logs Query Assist improvements reduce the complexity of Sumo Logic’s query language, making it easier to discover relevant fields, minimize errors, and benefit from intelligent query-building assistance.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::sumo Preview release
+This is a Preview release. To learn more, contact your Sumo Logic account executive. [Register here](https://docs.google.com/forms/d/e/1FAIpQLSdtmzRn1NyAdk1rXGZrJrpQQwR2i9FKOYd3uKLwEzrkZGVrwQ/viewform).
+:::
+
+Query Assist simplifies query-building in Sumo Logic by offering real-time syntax suggestions, schema prompts, and partial/full query predictions. These enhancements streamline the process for users at all skill levels, making it easier to write accurate and efficient queries.
+
+This feature reduces the complexity of query building, enabling easier discovery of relevant fields, minimizing errors, and providing intelligent assistance for query creation.
+
+<img src={useBaseUrl('img/search/get-started-search/query-assist-preview1.gif')} alt="query assist demo gif" width="700"/>
+
+## Key enhancements
+
+* **Simplified learning curve**. Syntax suggestions and schema prompts make it easy to write queries without extensive knowledge of query syntax.
+* **Enhanced auto-complete**. Context-aware suggestions improve accuracy and efficiency, adapting to your queries and organizational patterns.
+* **Streamlined field discovery**. Relevant fields are automatically suggested, especially for structured data like JSON, eliminating manual searching.
+
+## How to get started
+
+1. **Start a log search**. From the [**Classic UI**](/docs/get-started/sumo-logic-ui-classic) or the [**New UI**](/docs/get-started/sumo-logic-ui), go to **Log Search** to start typing your query.
+2. **Leverage syntax and schema suggestions**. As you type, watch for real-time suggestions for fields and operators. Select options using your arrow keys and press `Tab` to auto-complete.
+3. **Take advantage of predictions**. Use partial and full query predictions to build queries faster and more accurately.
+4. **Review and correct errors**. Correct flagged errors before running your query.
+
+## Features
+
```

Comment:
Syntax Suggestion

Highlight how as user types the first letters of a particular operator, all matching operator related to that letter/word are offerred as suggestion